src/collections/_documentation/workflow/releases/index.md

```diff
@@ -334,3 +334,74 @@ You can also use our [API]({%- link _documentation/api/releases/post-release-dep
 ## Release Artifacts
 
 JavaScript and iOS projects can utilize release artifacts to unminify or symbolicate error stack traces. To learn more, please check out our [iOS]({%- link _documentation/clients/cocoa/index.md -%}#sentry-cocoa-debug-symbols) and [JavaScript]({%- link _documentation/platforms/javascript/sourcemaps.md -%}) docs.
+
+## Release Health
+
+{% capture __alert_content -%}
+Sentry’s Release Health features are currently in beta. For more details about access to these features, feel free to reach out at [LINK: email?].
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Monitor the health of releases by observing user adoption, usage of the application, percentage of [crashes]({%- link _documentation/workflow/releases/release-health-glossary.md -%}#crash), and [session data]({%- link _documentation/workflow/releases/release-health-glossary.md -%}#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+### Releases Index
+
+The releases index page gives a high-level view of each release version, its associated project, the authors of each commit, the percentage of crash-free users, and the percentage of crash-free sessions. Each release links to one project. If a release has multiple projects, Sentry will duplicate the release data in relation to each unique project. The data will present adoption of the release and usage in the most recent 24 hours to 14 days.
+
+[ SCREENSHOT: Releases index page ]
+
+### Release Details
+
+The release details page focuses on an individual release. Elements of the release are presented in visual trends for crashes and sessions, specifics regarding each issue, and commit author breakdowns.
+
+[ SCREENSHOT: Releases detail page ]
+
+For more information about Sentry's Mobile features, see the full documentation for [Android SDK]({%- link _documentation/platforms/android/index.md -%}) and [iOS SDK](docs.sentry.io/platforms/cocoa/).
+
+### Session Data
```

Comment:
I would probably skip this section and provide this link only at the end of the page in the section related pages.

Reply:
Can we replace it with the getting started section?

Getting started
Currently the release health metrics are supported by the Android and IOS SDKs.
To benefit from the metrics you need to use version of the SDK that support the health metrics.
To learn more about the functionality in the Android SDK please check <Link>
Once your application is going to use SDK that sends the mobile data you will be able to see the mobile health data 
automatically on the release index page and release detail page.